### PR TITLE
Iteration3 favoriting deleting ideas

### PR DESCRIPTION
**Pull request by jskomal**
# Pull Request Template

## Description

This pull includes changes to functionality regarding starring and deleting cards as well as updating view in the DOM, as well as their object properties.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Functionality was tested in the browser repeatedly.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [n/a] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [n/a] I have added tests that prove my fix is effective or that my feature works
- [n/a] New and existing unit tests pass locally with my changes
- [n/a] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] We had fun

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -37,19 +40,56 @@ function hide(element) {
...
+      if (targetID === ideas[i].id.toString()) {
+        ideas[i].isStarred = !ideas[i].isStarred;
+        updateCardView();
+      }
+    }
+  }
+}
+
+function updateCardView() {
+  var emptyHTML = '';
```
Comment (CHaiz15) [inline main.js:86]: Is this necessary?

**Review by CHaiz15 — APPROVED** [review]
Looks great!

File: main.js (hunk 1 above)
Comment (zliibbe) [reply main.js:86]: No, it isn't. We'll fix that in iteration 4 pull request.

**Comment by zliibbe** [comment]
Merged live on zoom call with @c-o-de and @jskomal .

**Pull request merged**